helloworld/README.md

```diff
@@ -0,0 +1,10 @@
+## Synopsis
```

Comment:
Remember to read the **Documentation** section of the lab and write things in your README per the instructions.